packages/design-system/src/components/analytics/getAnalyticsContentFromRefs.ts

```diff
@@ -1,9 +1,6 @@
 import { RefObject } from 'react';
 
-export function getAnalyticsContentFromRefs(
-  refs: RefObject<any>[],
-  componentName?: string
```

Comment:
This arg isn't used in the function.

Reply:
Looks like it isn't used anymore, but there is a lingering instance of it being called with the second parameter that needs to be updated. Right now it fails to build.

Reply:
Somehow I missed that! I must have done the build before I saved this file. I really do appreciate that the project has a thorough set of unit and snapshot tests, it makes it much easier for newbs like me to change things and know if we're breaking something.